src/main/java/com/transferwise/openbanking/client/configuration/AspspDetails.java

```diff
@@ -261,4 +261,18 @@ default boolean detachedSignatureUsesDirectoryIssFormat() {
     default boolean registrationRequiresLowerCaseJtiClaim() {
         return false;
     }
+
+    /**
+     * Get the value to use as the intended issuer claim within a client request to get an access token.
+     *
+     * @return the issuer related to the jwtClaim.
+     */
+    String getIssuer();
+
+    /**
+     * Get the value to use as the audience issuer claim within a client request to get an access token.
+     *
+     * @return the audience related to the jwtClaim.
+     */
+    String getAudience();
```

Comment:
not sure what's the best name, there are already a Issuer and Audience property. See `getRegistrationAudience()` and `getRegistrationIssuer()` above

Reply:
We don't need the get `getAudience()` as we already have `getPrivateKeyJwtAuthenticationAudience()`